app/Player.js

```diff
@@ -35,31 +38,34 @@ function Player() {
             cross.scale.set(3.0, 3.0, 1.0);
             ship.add(cross);
         },
-        updateParticleValues: function () {
+        update: function () {
```

Comment:
Better name updateParticleValues?
As you only update the particle things. Not the player.